src/prng/prng.lisp

```diff
@@ -27,27 +20,42 @@
 
 (defmethod make-prng :around (name &key (seed :random))
   (let ((prng (call-next-method)))
-    (cond
-      ((eq seed nil))
-      ((find seed '(:random :urandom)) (read-os-random-seed seed prng))
-      ((or (pathnamep seed) (stringp seed)) (read-seed seed prng))
-      ((typep seed 'simple-octet-vector)
-       (reseed (slot-value prng 'generator) seed)
-       (incf (slot-value prng 'reseed-count)))
-      (t (error "SEED must be an octet vector, pathname indicator, :random or :urandom")))
+    (unless (eq name :fortuna-generator)
```

Comment:
A `FORTUNA-GENERATOR` cannot be reseeded at start; it must be initially seeded once before it may be reseeded.  This type-specific check is really ugly.